crates/aptos-faucet/migration/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "aptos-faucet-migration"
+description = "Migrations for the DB-based Checker for the Aptos faucet"
+version = "2.0.0"
+
+# Workspace inherited keys
+authors = { workspace = true }
+edition = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+publish = { workspace = true }
+repository = { workspace = true }
+rust-version = { workspace = true }
+
+[lib]
+path = "src/lib.rs"
+
+[dependencies]
+async-std = { version = "^1", features = ["attributes", "tokio1"] }
+sea-orm-migration = { version = "0.11.0", features = ["sqlx-postgres", "runtime-tokio-rustls"] }
```

Comment:
Is there a reason why we're using yet another ORM for this rather than using diesel?

Reply:
Because Max / Bowen recommended I try this instead of Diesel and I indeed found it better. Because of the feature this won't get pulled in the normal case, so it isn't too painful to have.

The SQL based checker is not actually used in prod btw, but you never know if we'll want it again at some point and it's perfectly functional code, so I figure it's worth keeping (behind a feature).